README.md

```diff
@@ -70,7 +70,7 @@ If additional help or guidance on using Apache Daffodil™, Apache Daffodil™ E
 
 # Community Feedback
 
-[![image 5](images/asf_logo_url.png)](https://www.apache.org)
+<img src="images/asf_logo_wide.png" width="581" alt="Apache Software Foundation Logo">
```

Comment:
A width of 581 seems pretty big. Is it worth making it smaller? No strong preference either way.